catroid/src/main/AndroidManifest.xml

```diff
@@ -38,6 +38,7 @@
     <uses-permission android:name="android.permission.GET_ACCOUNTS" />
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
     <uses-permission android:name="android.permission.RECORD_AUDIO" />
+    <uses-permission android:name="android.permission.REORDER_TASKS" />
```

Comment:
This permission allows the app to reorder applications on the android activity stack. For devices using an API version below ~ 5.0 (not sure about the exact version) this is necessary in order to put pocket paint to the correct position on the stack. The permission is not considered a (great risk)[https://developer.android.com/guide/topics/permissions/normal-permissions.html] by Google, so I'm sure we'll be fine using it.